src/components/DetailsPanel/DetailsPanel.js

```diff
@@ -1,8 +1,22 @@
+import { AboutVisualizationUnit } from '@dhis2/analytics'
+import PropTypes from 'prop-types'
 import React from 'react'
+import { connect } from 'react-redux'
+import { sGetCurrent } from '../../reducers/current'
 import classes from './styles/DetailsPanel.module.css'
 
-export const DetailsPanel = () => (
+export const DetailsPanel = ({ visualization }) => (
     <div className={classes.panel}>
-        <span style={{ color: 'red' }}>Details and interpretations panel</span>
+        <AboutVisualizationUnit visualization={visualization} />
```

Comment:
Instead of passing in the whole "visualization" (i.e. `current`), could we pass in only the props that are needed? (`const AboutVisualizationUnit = ({ type, id }) => {` -> `type` and `id`?)

Reply:
Yes, I actually did this change already, but I didn't push.